.github/workflows/E2E-chaoscenter-pipeline.yml

```diff
@@ -0,0 +1,82 @@
+name: ChaosCenter E2E testing
+
+on:
+  push:
+    branches:
+      - main  
+  pull_request:
+    branches:
+      - main 
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Minikube
+        run: |
+          sudo apt update && sudo apt install -y curl
+          curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+          sudo install minikube-linux-amd64 /usr/local/bin/minikube
+          minikube start
+
+      - name: Install kubectl
+        run: |
+          sudo apt update && sudo apt install -y kubectl
+      
+      - name: Install helm
+        run: |
+          curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+          chmod 700 get_helm.sh
+          ./get_helm.sh
```

Comment:
We won't need the setup steps.

`minikube`, `helm` & `kubectl` are already available on github action VMs!
Ref - https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md